prediction/sklearn/structured/custom_routines/README.md

```diff
@@ -0,0 +1,43 @@
+# Online Prediction with scikit-learn and Custom Routines
+
+## Overview
+
+In this tutorial, we go over the steps to deploy a model trained with scikit-learn to AI-Platform.
+We will be using the model that was trained using custom routines. We will deploy the model and pass
```

Comment:
Does a tutorial for this already exist? Can we link off to it (if on ai-platform-samples)?

Reply:
Never mind, saw the pre-req